platform/features/imagery-timeline/src/controllers/ImageryTimelineController.js

```diff
@@ -0,0 +1,79 @@
+// To do:
+//    use moment to format timestamp
+//    seperate functions for loading history and realtime imagery
+//    wait to display images until after load
+
+define(
+  ['moment'], 
+  function (moment) {
+     
+    function ImageryTimelineController($scope, openmct) {
+        $scope.images = [];
+        this.$scope = $scope;
+        this.openmct = openmct;
+        this.date = "";
+        this.time = "";
+        this.zone = "";
+        this.imageUrl = "";
+        this.history = {};
+        
+        this.subscribe = this.subscribe.bind(this);
+        this.updateValues = this.updateValues.bind(this);
+        
+        // Subscribes to telemetry when domain objec is available
+        this.subscribe(this.$scope.domainObject);
+        this.$scope.$on("$destroy", this.stopListening);
+        
+    }
+
+    ImageryTimelineController.prototype.subscribe = function (domainObject) {
+        this.date = "";
+        this.imageUrl = "";
+        this.openmct.objects.get(domainObject.getId())
+            .then(function (object) {
+                this.domainObject = object;
+                var metadata = this.openmct
+                    .telemetry
+                    .getMetadata(this.domainObject);
+                var timeKey = this.openmct.time.timeSystem().key;
+                this.timeFormat = this.openmct
+                    .telemetry
+                    .getValueFormatter(metadata.value(timeKey));
+                this.imageFormat = this.openmct
+                    .telemetry
+                    .getValueFormatter(metadata.valuesForHints(['image'])[0]);
+                this.unsubscribe = this.openmct.telemetry
+                    .subscribe(this.domainObject, this.updateValues);
+                this.openmct.telemetry
+                    .request(this.domainObject, {
+                        strategy: 'all',
+                        start: Date.now(),
+                        end: Date.now() + 90000 // for testing purposes, gets full image
```

Comment:
Whoa, it's probably a mistake that the example imagery provider will give you imagery that's in the future.

instead of hard coding time, use the time conductor, e.g. `this.openmct.telemetry.request(this.domainObject, this.openmct.time.bounds());`

You'll also need to listen to time conductor bounds changes and query for new data.